modules/nf-core/grabix/check/main.nf

```diff
@@ -0,0 +1,30 @@
+process GRABIX_CHECK {
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/grabix:0.1.8--hdcf5f25_9':
+        'biocontainers/grabix:0.1.8--hdcf5f25_9' }"
+
+    input:
+    path(input)
```

Comment:
```suggestion
    tuple val(meta), path(input)
```

nf-core pipelines usually use meta fields alongside their files to keep track of which file it is